models/action.go

```diff
@@ -191,6 +194,38 @@ func (a *Action) GetRepoLink() string {
 	return "/" + a.GetRepoPath()
 }
 
+// GetCommentLink returns link to action comment.
+func (a *Action) GetCommentLink() string {
+	if a == nil {
+		return "#"
+	}
+	if a.Comment == nil && a.CommentID != 0 {
+		a.Comment, _ = GetCommentByID(a.CommentID)
+	}
+	if a.Comment != nil {
+		return a.Comment.HTMLURL()
+	}
+	if len(a.GetIssueInfos()) == 0 {
+		return "#"
+	}
+	//Return link to issue
+	issueIDString := a.GetIssueInfos()[0]
+	issueID, err := strconv.ParseInt(issueIDString, 10, 64)
+
+	if err != nil {
+		return "#"
+	}
+
+	issue, err := GetIssueByID(issueID)
+
+	if err != nil {
+		return "#"
+	}
+
+	return issue.HTMLURL()
+
```

Comment:
Please remove unneeded newline after last return and before both `if err != nil {`